test/helpers/fixtures.js

```diff
@@ -1,49 +1,83 @@
 const	withPrimaryNav = `
 	<header class='o-header-services' data-o-component='o-header-services'>
-		<div class='o-header-services__top o-header-services__container'>
+		<div class='o-header-services__top'>
 			<div class="o-header-services__hamburger">
-				<a class="o-header-services__hamburger-icon" href="#" aria-controls="o-header-drawer" role="button" data-o-toggle--js="true" aria-expanded="false"><span class="o-header__visually-hidden">Menu</span></a>
+				<a class="o-header-services__hamburger-icon" href="#" role="button"><span class="o-header__visually-hidden">Menu</span></a>
 			</div>
 			<div class='o-header-services__logo'></div>
 			<div class='o-header-services__title'>
-				<h1 class='o-header-services__product-name'><a href="/">Tool or Service name</a></h1><span class='o-header-subrand__product-tagline'>Tagline to explain the product here</span>
+				<h1 class='o-header-services__product-name'>
+					<a href="/">Tool or Service name</a>
+				</h1>
+				<span class='o-header-services__product-tagline'>Tagline to explain the product here</span>
 			</div>
 			<ul class="o-header-services__related-content">
-				<li class="o-header-services__related-content-list-item">
-					<a class="o-header-services__related-content-link" href="#">XXXX</a>
+				<li>
+					<a href="#">XXXX</a>
 				</li>
-				<li class="o-header-services__related-content-list-item">
-					<a class="o-header-services__related-content-link" href="#">Sign in</a>
+				<li>
+					<a href="#">Sign in</a>
 				</li>
 			</ul>
 		</div>
 		<nav class='o-header-services__primary-nav'>
-			<div class='o-header-services__nav-container'>
-				<ul class='o-header-services__primary-nav-list'>
-					<li class='o-header-services__primary-nav-item'>
-						<a class='o-header-services__primary-nav-link o-header-services__primary-nav-link--selected' href='#item-1'>
-							Nav item 1
-						</a>
-					</li>
-					<li class='o-header-services__primary-nav-item'>
-						<a class='o-header-services__primary-nav-link' href='#'>
-							Nav item 2
-						</a>
-					</li>
-					<li class='o-header-services__primary-nav-item'>
-						<a class='o-header-services__primary-nav-link' href='#'>
-							Nav item 3
-						</a>
-					</li>
-					<li class='o-header-services__primary-nav-item'>
-						<a class='o-header-services__primary-nav-link' href='#'>
-							Nav item 4
-						</a>
-					</li>
-				</ul>
-				<button class="o-header-services__scroll-button o-header-services__scroll-button--left" aria-hidden="true" disabled></button>
-				<button class="o-header-services__scroll-button o-header-services__scroll-button--right" aria-hidden="true" disabled></button>
-			</div>
+			<<ul class="o-header-services__primary-nav-list">
+				<li data-o-header-services-level="1">
+					<a href="#">Fruit</a>
+					<ul data-o-header-services-level="2">
```

Comment:
Accessibility needs a little more work. The submenus are present at all times, and there's no indication that clicking the top level nav should open them up. Sorry if I misunderstood your `aria-expanded` chat earlier.

Reply:
Gah, it doesn't look like there are great options though. 

I wonder if we can speak to Tom about making the link a link, and make the arrows a button which opens the subnav, similar to o-header.
<img width="311" alt="screenshot 2019-01-23 at 17 16 47" src="https://user-images.githubusercontent.com/10405691/51624544-f756d680-1f32-11e9-9026-54fc7a9263e4.png">